sdk/search/azure-search/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version><!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../../pom.client.xml</relativePath>
+  </parent>
+
+  <name>Microsoft Azure Search client for Java</name>
+  <description>This package contains client functionality for Microsoft Azure Search</description>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-search</artifactId>
+  <version>1.0.0-beta.2</version> <!-- {x-version-update;com.azure:azure-search;current} -->
+  <packaging>jar</packaging>
+
+  <properties>
+    <project.automatic.module.name>com.azure.search</project.automatic.module.name>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+  </properties>
+
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
```

Comment:
should inherit the version from parent.
Is there a strict need for this specific version ?

Reply:
no need for this specifc version. Updating with parent version

Reply:
@navalev this particular plugin entry and version are fine.
@g2vinay the version entry is required for every dependency and every plugin.